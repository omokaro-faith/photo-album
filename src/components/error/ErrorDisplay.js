import React from 'react';
import { PropTypes } from 'prop-types';

const ErroDisplay = ({item}) => (
  <div className='alert'>
    <div>{`Cannot fetch ${item}, there might be an issue with your network connection`}</div>
  </div> 
)

ErroDisplay.propTypes = {
  item: PropTypes.string,
}


 export default ErroDisplay; 