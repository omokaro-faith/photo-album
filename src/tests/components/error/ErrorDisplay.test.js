import React from 'react';
import { shallow } from 'enzyme';
import ErrorDisplay from '../../../components/error/ErrorDisplay';

describe('ErrorDisplay', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ErrorDisplay/>);
  });

  afterEach(() => { wrapper = null; });

  
  describe('renders', () => {
    it('should exist', () => {
      expect(wrapper.exists()).toBe(true);
    });

    it('should not have any regressions', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
