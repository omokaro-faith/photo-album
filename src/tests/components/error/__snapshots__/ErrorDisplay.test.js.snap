// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorDisplay renders should not have any regressions 1`] = `ShallowWrapper {}`;
